### ISSUE TITLE
Table properties are not editable when we maximise the editor

### ISSUE THREAD

**Issue by vipulnarang95** [django-cms/djangocms-text-ckeditor#677]
When we add a table in text field, the table properties are not editable when we maximise the editor.

**Comment by fsbraun** [comment]
Can't reproduce. Try giving more details: what is not editable, what did you do before... Screenshot?

**Comment by vipulnarang95** [comment]
Hi @fsbraun , apologies for replying so late. Steps to reproduce:
Add a text plugin to a page> add a table to the body> right click on any coloumn, you'll be able to see the properties and options on table > 
Expand body section to full screen and again try to right click, the menu doesn't appears.
![Uploading 359775295...]
![Uploading 359775380...]

In the second ss of full screen the table properties doesn't appear.

**Comment by vipulnarang95** [comment]
@fsbraun Any thoughts on this?

**Comment by vipulnarang95** [comment]
@fsbraun can you please check this?

**Comment by fsbraun** [comment]
@vipulnarang95 Yes, I did. And I cannot reproduce the issue. :-( Any custom or non-standard CKEditor4 plugins?

Also, I notice that in your first example you're editing an `HTMLField` and in the second example a text plugin. They might have different configurations in your project setup. Can you check?

Here are my screenshots:
## Regular

<img width="696" alt="image" src="https://github.com/user-attachments/assets/f7b6edc9-90e0-4f83-b939-79004fc33629">

## Maximized

<img width="753" alt="image" src="https://github.com/user-attachments/assets/42e53a4f-819a-4b06-8ad2-5a571d0d667a">

**Comment by fsbraun** [comment]
Final thought: Your dropdown menu is somewhat too large in your first screenshot, and seems to have zero height in your second screenshot. Any CSS at work that interferes with the height?

**Comment by vipulnarang95** [comment]
Let me check and get back to you!

**Comment by vipulnarang95** [comment]
@fsbraun  yes we have a custom plugin built on top of text plugin.
Also the second screenshot is the maximized of the first one, it is the single component and the issue is still there? 
Any other thoughts or suggestion?

**Comment by FinalAngel** [comment]
Hi @fsbraun, a bit more context to the problem. The editor is added as a custom `HTMLField` (not the general text plugin). The issue only occurs then. I also created a brief screencast:

![2024-10-15 11 09 42](https://github.com/user-attachments/assets/c0bcfd7f-b3e9-4777-9bbc-a0e346b64eda)

**Comment by vipulnarang95** [comment]
Hi @fsbraun, can you please see the screen capture? It shows the exact issue!

**Comment by fsbraun** [comment]
@vipulnarang95 Still cannot reproduce this behavior with current djangocms-text-ckeditor 5.1.6, neither for a Text plugin nor for a `HTMLField`. Can you check what CSS rule defines the height of the dropdown?

**Comment by fsbraun** [comment]
Closing this issue since djangocms-text-ckeditor is now deprecated and replaced by djangocms-text.
